user_guide_src/source/changelogs/v4.5.8.rst

```diff
@@ -31,6 +31,7 @@ Bugs Fixed
 **********
 
 - **Database:** Fixed a bug where ``Builder::affectedRows()`` threw an error when the previous query call failed in ``Postgre`` and ``SQLSRV`` drivers.
+- **Security** Fixed a bug where the CSRF token validation could fail on malformed input, causing a generic 500 error instead of handling the input gracefully.
```

Comment:
```suggestion
- **Security:** Fixed a bug where the CSRF token validation could fail on malformed input, causing a generic HTTP 500 status code instead of handling the input gracefully.
```